src/main/java/Tasks/task.java

```diff
@@ -0,0 +1,78 @@
+package Tasks;
+
+import java.time.LocalDateTime;
+
+/**
+ * Represents a task that will be created by the user to be added into the list.
+ */
+public class task {
```

Comment:
Hey, class names should be written in PascalCase so the first letter should be capitalized here.